_projects/2024/100540803/100540803.Rmd

```diff
@@ -0,0 +1,601 @@
+---
+title: "Global Fish Stock Overfishing Status"
+description: "Replication of a chart illustrating fish capture and abundance relative to sustainable levels."
+author: 
+  name: "Linghan Zheng"
+  date: "Dec.24,2024"
+format: html
+editor: visual

```

Comment:
Please, set the header as indicated in the [tutorial](https://csslab.uc3m.es/dataviz/tutorials/project/#adding-your-own-article), where it says "the header should look like this"...